python/lsst/ts/linearstage/csc.py

```diff
@@ -224,7 +244,9 @@ async def handle_summary_state(self):
                     )
                 except RuntimeError as e:
                     err_msg = "Failed to establish connection to component"
-                    await self.fault(code=2, report=f"{err_msg}: {e}")
+                    await self.fault(

```

Comment:
I am not sure if I agree with this way of calling `self.fault`. I will review this after you have made the requested changes.